src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,33 @@
+.stars__star {
+  display: flex;
+  place-content: center;
+  align-items: center;
+  width: 16px;
+  height: 16px;
+
+  &:not(:last-child) {
+    margin-right: 4px;
+  }
+}
+
+.stars {
+  display: flex;
+  align-content: center;
```

Comment:
The `align-content` property is not necessary here. `align-content` determines the spacing between lines, while `align-items` determines how the items as a whole are aligned within the container. Since it is expected that `.stars` will be a single-row flex container, `align-content` can be removed.